intake_stac/catalog.py

```diff
@@ -331,15 +331,25 @@ def __init__(self, key, item, stacked=False):
     def _get_driver(self, entry):
         drivers = {
             'application/netcdf': 'netcdf',
+            'application/x-netcdf': 'netcdf',
+            'application/parquet': 'parquet',
+            'application/x-parquet': 'parquet',
+            'application/x-hdf5': 'netcdf',
+            # 'application/x-hdf': '',
             'image/vnd.stac.geotiff': 'rasterio',
             'image/vnd.stac.geotiff; cloud-optimized=true': 'rasterio',
```

Comment:
should these be removed in favor of the new tiff labels below? or are they still valid?

Reply:
I chose to not remove and previously used tags that might still exist in STAC catalogs, they could be marked for future deprecation. The new COG tag isn't official yet either, but community adopted for now.